hack/generate-pull-secret-entries.py

```diff
@@ -0,0 +1,117 @@
+#!/usr/bin/env python3
```

Comment:
Can we make this something that just validates that all .dockerconfigjson for `cluster-pul-secret` entries are identical except for the local cluster? This now rewrites the whole config

Reply:
What is wrong with the current approach, specifically? It's not *only* that they need to be identical, they *also* must have correct content.

Reply:
We auto-generate and re-format every other file in this repo, I can't see how another is any worse (with a convenient `make` target to do it, too)